packages/vats/src/provisionPoolKit.js

```diff
@@ -378,6 +382,24 @@ export const prepareProvisionPoolKit = (
               },
             },
           );
+
+          if (metrics) {
+            // Restore state, but don't publishMetrics()
+            // because that already happened in the last incarnation.
```

Comment:
Hm. now it's not just a misleading comment. It's incorrect. We're not doing a zoe contract upgrade between incarnations here. We're restarting the swingset VM. So the metrics should be published right away in the first incarnation.

It's not the end of the world, since the problem goes away the next time anyone provisions a smart wallet.